qiskit/tools/events/progressbar.py

```diff
@@ -118,6 +118,25 @@ class TextProgressBar(BaseProgressBar):
 
     output_handler : the handler the progress bar should be written to, default
                      is sys.stdout, another option is sys.stderr
+
+    .. jupyter-execute::
+
+       # example 1
```

Comment:
It would probably be nicer to group this as:

```
Examples:
    
    The progress bar can be used to track the progress of a `parallel_map`

    .. jupyter-execute::
   
       ....

    And it can also be used individually:

    .. juypter-execute::

        ...
```